pkg/controller/compliancescan/compliancescan_controller.go

```diff
@@ -747,6 +752,22 @@ func gatherResults(r *ReconcileComplianceScan, instance *compv1alpha1.Compliance
 		}
 	}
 
+	// If there are any inconsistent results, always just return
+	// the state as inconsistent unless there was an error earlier
+	var checkList compv1alpha1.ComplianceCheckResultList
+	checkListOpts := client.MatchingLabels{
+		compv1alpha1.ComplianceCheckInconsistentLabel: "",
+		compv1alpha1.ScanLabel:                        instance.Name,
```

Comment:
This was missing, thus any inconsistent scan in the namespaces marked all subsequent scans as inconsistent.